rfcs/text/0030/output.yml

```diff
@@ -0,0 +1,48 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+- name: output
+  title: Output
+  group: 2
+  short: Fields describing a chunk of tty output generated by a process.
+  description: >
+    Fields describing a chunk of tty output generated by a process.
+
+    Typically these fields will be set when event.action: output and event.category: process
+  type: group
+
+  fields:
+    - name: fd
```

Comment:
If you're intending to use the file_description.* fieldset here, you'll want to remove this and instead add a reuse to file_description

```
- name: fd
  reusable:
    expected:
      - at: output
        as: fd
```